Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ruby:2.3
+FROM ruby:2.4
```

Comment:
I would rather this be ruby:2.4.0.  I had trouble with the existing Dockerfile because my cached ruby:2.3 was 2.3.1 which didn't match the ruby specified in Gemfile.  We won't have that problem if we use same granularity in Gemfile and Dockerfile.